pkg/apis/storage/v1/util/helpers.go

```diff
@@ -24,6 +24,10 @@ import (
 // marks a class as the default StorageClass
 const IsDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-class"
 
+// IsWindowsStorageClassAnnotation represents a StorageClass annotation that
+// marks a class as the Windows StorageClass
+const IsWindowsStorageClassAnnotation = "storageclass.kubernetes.io/is-windows-class"
```

Comment:
This implies that StorageClass for windows will not work on Linux or vice versa. Are we sure it's not possible? What if we name it to something like "windows-compatible-class"?

Also, should it be a label or an annotation? Do we anticipate this information being useful for end users?

We need to be extra careful with adding new annotations or labels, as they are now [considered API](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-parts-of-the-api), and we cannot remove them until v2. Maybe we need to make it an alpha annotation, although that convention is also [deprecated](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions)...

Reply:
This information is mainly for system to find out a storageclass what works for Windows, from this point of view, I think annotation is good. 
For users, the fsType is a field they can check directly. I think this window-compatible information seems not very necessary.
I think "windows-compatible-storageclass" is good.